src/main/java/com/puppycrawl/tools/checkstyle/checks/TrailingCommentCheck.java

```diff
@@ -102,23 +102,34 @@
  * &lt;module name=&quot;TrailingComment&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * // OK
+ * if (&#47;&#42; OK &#42;&#47; x &#62; 5) {}
+ * int a = 5; // violation, string before comment should match pattern "^[\s});]*$"
```

Comment:
regexp that we use by default is very cryptic, lets use 
`// violation`

Reply:
Done! Applies for the other requested change as well.
![Screenshot from 2022-06-19 20-39-54](https://user-images.githubusercontent.com/23459549/174495851-4d7837d8-1951-4740-af1a-e7badb4d0a4d.png)